src/components/v2/appDetails/sourceInfo/scaleWorkloads/ScaleWorkloadsModal.component.tsx

```diff
@@ -233,12 +234,16 @@ export default function ScaleWorkloadsModal({ appId, onClose, history }: ScaleWo
         const _nameSelectionKey = isHibernateReq ? 'scaleDown' : 'restore'
 
         try {
+            let payloadAppId = appId;
+            if(appDetails.appType != AppType.EXTERNAL_HELM_CHART){
```

Comment:
This is intentional right? We are doing this only for helm & devtron apps

Reply:
yes